utils/dependency_management.py

```diff
@@ -77,6 +78,24 @@ def _setuptools_to_conda(req):
     return Requirement.parse(str(req))
 
 
+class _Entry:
```

Comment:
instead of all these underscores, one could also just use  `__all__` to expose just the classes/functions you want to expose

Reply:
This is a stand-alone script, so there is not anything exposed. The underscores are merely used to guide the reader's eye. Rather than introducing `__all__`, we can also remove all underscores.